apps/api/src/app/patient/patient.controller.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Param,
+  Delete,
+  Put,
+} from '@nestjs/common';
+import { PatientService } from './patient.service';
+import { CreatePatientDto } from './dto/create-patient.dto';
+
+@Controller('patient')
+export class PatientController {
+  constructor(private readonly patientService: PatientService) {}
+
+  @Post('/')
+  create(@Body() createPatientDto: CreatePatientDto) {
+    return this.patientService.create(createPatientDto);
+  }
+
+  @Get('/:name')
+  findEmployeeByName(@Param('name') name) {
+    return this.patientService.findEmployeeByName(name);
```

Comment:
Employee?

Reply:
please check this @chxru @Madhurapabu @narangodaNPP